app-ios/README.md

```diff
@@ -11,7 +11,8 @@ cd ../
 ```
 
 ## Requirements
-- Xcode 13.4.1 ([Workaround needed for build](#workaround))
+- Xcode 13.4.1
+⚠️ If you cloned for the first time, you need to try some workaround. Please check it. [Workaround needed for build](#workaround)
```

Comment:
I remember GitHub has a special syntax for markdown.
How about apply this? 👀 

```suggestion
  > **Warning**
  > If you cloned for the first time, you need to try some workaround. Please check it. [Workaround needed for build](#workaround)
```

https://github.com/community/community/discussions/16925

Reply:
Thanks!